### ISSUE TITLE
npm install -g @graphprotocol/graph-cli AND yarn global add @graphprotocol/graph-cli are failing because of 'feat/smaller' being unavailable.

### ISSUE THREAD

**Issue by br0wnD3v** [graphprotocol/graph-tooling#1264]
### Which packages are impacted by your issue?

@graphprotocol/graph-cli

### Describe the issue

Using npm version `9.5.0` and yarn `1.22.19` 

The npm command is failing as 
```
npm ERR! code 1
npm ERR! The git reference could not be found
npm ERR! command git --no-replace-objects checkout feat/smaller
npm ERR! error: pathspec 'feat/smaller' did not match any file(s) known to git

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2023-04-09T20_32_01_402Z-debug-0.log
```

And the yarn command is failing as 
```
error Couldn't find match for "feat/smaller" in "refs/heads/1.3.x,refs/heads/1.4.x,refs/heads/master,refs/tags/v1.3.2,refs/tags/v1.4.10,refs/tags/v1.4.11,refs/tags/v1.4.7,refs/tags/v1.4.8,refs/tags/v1.4.9,refs/tags/v1.5.0,refs/tags/v1.5.1,refs/tags/v1.5.2,refs/tags/v1.6.0,refs/tags/v1.6.1,refs/tags/v1.6.2,refs/tags/v2.0.0" for "https://github.com/hugomrdias/concat-stream.git".
info Visit https://yarnpkg.com/en/docs/cli/global for documentation about this command.
``` 
being the main reason.

Would appreciate any solution to this. Thanks : )

### Reproduction

bash terminal

### Steps to Reproduce the Bug or Issue

$npm install -g @graphprotocol/graph-cli
OR
$yarn global add @graphprotocol/graph-cli

### Expected behavior

Should install the package @graphprotocol/graph-cli globally and allow the user call `graph` in the terminal.

### Platform

- OS: Ubuntu
- NodeJS: v18.15.0
- `@graphprotocol/graph-cli` version(s): 0.45.0(The installation of older versions is failing too)

**Comment by 0xchetan** [comment]
The same thing is happening to me. I've tried copying existing `node_modules` from my older subgraph folders, but the syncing seems to be failing.

**Comment by mihirgupta0900** [comment]
@br0wnD3v @0xchetan This was supposed to be fixed by https://github.com/graphprotocol/graph-tooling/pull/1263 but doesn't fix for some reason. It will likely be fixed by https://github.com/graphprotocol/graph-tooling/pull/1268

Until then you can do the following changes to your root `package.json`. This works for the time being, recommend to upgrading to the latest version when https://github.com/graphprotocol/graph-tooling/pull/1268 is merged:
```json
 "pnpm": {
    "overrides": {
      "ipfs-http-client@34.0.0>concat-stream": "github:mihirgupta0900/concat-stream#master"
    }
  }
```

**Comment by tomwhiteman** [comment]
Same issue. @mihirgupta0900 's solution doesn't work for me either.

**Comment by ronin66666** [comment]
The same thing is happening to me.

**Comment by tomwhiteman** [comment]
@mihirgupta0900 's solution works when installing the cli locally. However, when accessing it and using it to initialize the subgraph (with npx), it fails again with the same error. 

Please fix asap.

**Comment by paymog** [comment]
> @br0wnD3v @0xchetan This was supposed to be fixed by #1263 but doesn't fix for some reason. It will likely be fixed by #1268
> 
> Until then you can do the following changes to your root `package.json`. This works for the time being, recommend to upgrading to the latest version when #1268 is merged:
> 
> ```json
>  "pnpm": {
>     "overrides": {
>       "ipfs-http-client@34.0.0>concat-stream": "github:mihirgupta0900/concat-stream#master"
>     }
>   }
> ```

This solution unfortunatley doesn't work for me either even though I'm also using pnpm. It seems the override isn't being picked up by pnpm.

**Comment by saihaj** [comment]
you can also try to pin to old version `yarn add @graphprotocol/graph-cli@0.44.0`/`ppm install @graphprotocol/graph-cli@0.44` while I try to fix it in #1268

**Comment by br0wnD3v** [comment]
@saihaj  I tried what you said but the command keeps failing with the same reason

**Comment by br0wnD3v** [comment]
@mihirgupta0900 What version of @graphprotocol/graph-cli do you recommend to be added after adding the statement to package.json?

**Comment by 0xchetan** [comment]
> > @br0wnD3v @0xchetan This was supposed to be fixed by #1263 but doesn't fix for some reason. It will likely be fixed by #1268
> > Until then you can do the following changes to your root `package.json`. This works for the time being, recommend to upgrading to the latest version when #1268 is merged:
> > ```json
> >  "pnpm": {
> >     "overrides": {
> >       "ipfs-http-client@34.0.0>concat-stream": "github:mihirgupta0900/concat-stream#master"
> >     }
> >   }
> > ```
> 
> This solution unfortunatley doesn't work for me either even though I'm also using pnpm. It seems the override isn't being picked up by pnpm.
> 
> EDIT: ah, turns out I was using the override incorrectly given that I'm in a monorepo setup. This now works for me.

This worked for me.

**Comment by 0xchetan** [comment]
> @br0wnD3v @0xchetan This was supposed to be fixed by #1263 but doesn't fix for some reason. It will likely be fixed by #1268
> 
> Until then you can do the following changes to your root `package.json`. This works for the time being, recommend to upgrading to the latest version when #1268 is merged:
> 
> ```json
>  "pnpm": {
>     "overrides": {
>       "ipfs-http-client@34.0.0>concat-stream": "github:mihirgupta0900/concat-stream#master"
>     }
>   }
> ```

this worked for me.

**Comment by saihaj** [comment]
should be patched in [@graphprotocol/graph-cli@0.45.2](https://github.com/graphprotocol/graph-tooling/releases/tag/release-1681166300461)

**Comment by br0wnD3v** [comment]
Amazing work sir @saihaj Cheers 🥂
